### DISCUSSION TITLE
Compact monthly and weekly views on mobile

### DISCUSSION THREAD

**Discussion by notanovel** [davish/obsidian-full-calendar#559] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

Would it be possible to have compact monthly and weekly views of the calendar for mobile?

### ✔️ Solution

The solution to small mobile screens could be to show the entire month or week and just dots of the events (not the full description like on desktop or ipad) in the color that was selected for each calendar. That would show that "something" is happening on that day, then, clicking on a dot or changing view could be used to see the actual appointment/invite/note.

### ❓ Alternatives

The current best alternative that I can see is to use the "list" feature (instead of "3" or "1" entry).

**Comment by Pritolus** [comment]
Please add this. Month view is by far the most useful to me and I'm not really able to switch to Full Calendar without it. It's so strange to me that it's not a feature, but that could be down to how my brain works. The fact that it's still not a thing definitely makes it feel like it's not a priority 
Would it be possible for someone to make their own calendar view for Full Calendar maybe?
